misk/src/main/kotlin/misk/web/extractors/HeadersParameterExtractorFactory.kt

```diff
@@ -5,16 +5,22 @@ import misk.web.Request
 import misk.web.RequestHeaders
 import misk.web.actions.WebAction
 import java.util.regex.Matcher
+import javax.inject.Singleton
 import kotlin.reflect.KParameter
 import kotlin.reflect.full.findAnnotation
 
+@Singleton
```

Comment:
Does this do anything?

Reply:
Nope, apparently that's what happens when I write code at the end of the day. I'm surprised you can even annotate an instance as a singleton.

Reply:
Updated to be an object instance of an object assigned to a value after talking to @AlecStrong. This keeps behavior the same while making it searchable in Intellij.